ci/travis.rb

```diff
@@ -23,7 +23,8 @@ class Build
     'ar'       => 'activerecord',
     'av'       => 'actionview',
     'aj'       => 'activejob',
-    'guides'   => 'guides'
+    'guides'   => 'guides',
+    'ita'      => 'integration-test-app'
```

Comment:
Can we add a trailing comma here so there's not churn on this line in the future?